spec/support/ansible_shared/automation_manager/refresh_configuartion_script_source.rb

```diff
@@ -47,7 +49,11 @@
         configuration_script_payloads = configuration_script_source.configuration_script_payloads
 
         VCR.use_cassette(cassette_path) do
-          EmsRefresh.refresh([[configuration_script_source.class.to_s, configuration_script_source.id]])
+          # EmsRefresh.refresh([[configuration_script_source.class.to_s, configuration_script_source.id]])
```

Comment:
So if the class wasn't listed in https://github.com/Ladas/manageiq/blob/3227d2b7f73da39db5f6b52b079977e2e32312e9/app/models/ems_refresh.rb#L136 , I suppose it never worked.

Since the target wouldn't go through ```EmsRefresh.queue_refresh```, which is used in production workflow

Reply:
anyway, this change is needed there but brokes embedded ansible specs from core (this is shared example) so there will be cycle dependency